src/continuous-scroll.js

```diff
@@ -0,0 +1,202 @@
+/* global TweenLite, TimelineLite, Linear */
+import React from "react";
+import "gsap";
+
+let tween = null,
+  timeline = null;
+
+const ContinuousScroll = React.createClass({
+  propTypes: {
+    type: React.PropTypes.oneOf(["none", "continuous", "page"]),
+    speed: React.PropTypes.oneOf(["slow", "medium", "fast"]),
+    pause: React.PropTypes.number,
+    height: React.PropTypes.number.isRequired,
+    data: React.PropTypes.array.isRequired,
+    isRefreshing: React.PropTypes.bool,
```

Comment:
@donnapep I don't think the component should manage data either, so agreed on the data being a prop. I'm curious though, would the `isRefreshing` be set back to `false` at any point since this prop value is being set externally? How/when would that occur?